src/components/Grid/GridComponent.vue

```diff
@@ -76,6 +93,9 @@ export default Vue.extend({
         assessmentId: this.gridAssessments[colIndex].id
       })
     },
+    scrollUp () {
```

Comment:
this is never called right ? or am missing something , if it is never called it can be removed